### DISCUSSION TITLE
What's the theme being used in the preview image?

### DISCUSSION THREAD

**Discussion by noahcil** [IdreesInc/Monocraft#102] [Q&A] [answered]
Sorry, I know this isn't directly related to your (very well made) font, but I thought it looked good with the font and would like to use it myself.

**Comment by IdreesInc** [comment, marked as answer]
[Monokai Pro](https://monokai.pro/)! It costs money to use, but it is only ten dollars and also customizes the look of VSCode with new icons and colors. The free trial is perpetual (though it bugs you with a popup when you open up the IDE), but after using it for a few years I decided to buy it. There are free versions out there that look similar and only change the syntax highlighting, so check that out if spending ten dollars on a theme seems insane.
